src/renderer/i18n/index.tsx

```diff
@@ -40,7 +40,13 @@ export function LocaleProvider({ children }: React.PropsWithChildren<{}>) {
 
 export async function validateLocale(locale: string): Promise<boolean> {
   const languages = await tx.getLanguages();
-  return languages.some((lang: Record<"code", string>) =>
-    locale.startsWith(lang.code)
-  );
+  return languages.some((lang: Record<"code", string>) => {
+    let code = lang.code;
+    if (code === "pt_BR")
+    {
+      code = "pt-BR";
+    }
+
+    return locale.startsWith(code);
```

Comment:
IMO this problem will happen with other `Unix style tagging`. How about change this code like...
```suggestion
    return locale.startsWith(lang.code.replace('_', '-'));
```

Reply:
That's batter.😁